### PR TITLE
Add router that does nothing for bitswap_wo_routing test

### PR DESCRIPTION
**Pull request by heems**
@whyrusleeping 
I added a new nilrouting package and routing option that could be useful for future routerless tests.

License: MIT
Signed-off-by: Karthik Bala <karthikbala444@gmail.com>

### CODE REVIEW

**Comment by jbenet** [comment]
LGTM -- @whyrusleeping ?

File: core/core.go [hunk 1]
```diff
@@ -615,3 +616,4 @@ type RoutingOption func(context.Context, p2phost.Host, ds.ThreadSafeDatastore) (
 type DiscoveryOption func(p2phost.Host) (discovery.Service, error)
 
 var DHTOption RoutingOption = constructDHTRouting
+var NilOption RoutingOption = nilrouting.ConstructNilRouting
```
Comment (whyrusleeping) [inline core/core.go]: maybe call this the `NilRouterOption`.

**Comment by whyrusleeping** [comment]
One comment, otherwise LGTM

**Comment by jbenet** [comment]
LGTM

**Pull request merged**